galaxy_ng/app/api/v3/serializers/sync.py

```diff
@@ -0,0 +1,45 @@
+from galaxy_ng.app.constants import VALID_REMOTE_NAMES
+from pulp_ansible.app.models import CollectionRemote
+from rest_framework import serializers
+
+
+class SyncConfigSerializer(serializers.ModelSerializer):
+    created_at = serializers.DateTimeField(source='pulp_created')
+    updated_at = serializers.DateTimeField(source='pulp_last_updated')
+
+    class Meta:
+        model = CollectionRemote
+        fields = (
+            'name',
+            'url',
+            'auth_url',
+            'policy',
+            'requirements_file',
+            'created_at',
+            'updated_at',
+        )
+
+    def validate(self, data):
+        errors = {}
+        name = data['name']
+        requirements_file = data['requirements_file']
+
+        if name not in VALID_REMOTE_NAMES:
+            errors['name'] = (
+                f'{name} is not a valid remote name, possible values are {VALID_REMOTE_NAMES}'
+            )
+
+        if name == 'rh-certified' and requirements_file is not None:
+            errors['requirements_file'] = (
+                'The field requirements_file must be null for rh-certified repository'
+            )
+
+        if name == 'community' and requirements_file is None:
```

Comment:
Instead of checking the name of the remote, lets check that the URL contains `api/v2`, since that's the version that's supported on Galaxy.